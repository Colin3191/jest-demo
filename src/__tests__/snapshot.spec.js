const { generateDiv } = require("../dom");
require("../jsdom-config");

it("Dom快照测试", () => {
  generateDiv();
  expect(document.getElementsByClassName("c1")).toMatchSnapshot();
});
