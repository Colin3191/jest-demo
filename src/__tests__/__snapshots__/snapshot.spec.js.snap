// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dom快照测试 1`] = `
HTMLCollection [
  <div
    class="c1"
  />,
]
`;
